base_vat_optional_vies/README.rst

```diff
@@ -0,0 +1,82 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===================================
+Optional validation of VAT via VIES
+===================================
+
+This module extends base_vat module features allowing to know if VIES
+validation was passed or not.
+
+Then you can use "VIES validation passed" field in order to show VAT ID with
+or without country preffix in invoices, for instance.
+
+
+Installation
+============
+
+To install this module, you need vatnumber python library:
+
+* pip install vatnumber
+
+
+Configuration
+=============
+
+In order to activate VIES validation, you must set this option in your company:
+Settings > Companies > Companies > Your Company > Configuration > Accounting > VIES VAT Check
+
+When VIES VAT Check is activated:
+
+* Odoo will try to validate VAT against VIES online service
+* If passed, then "VIES validation passed" field will be True
+* If not passed, then try to validate using country validation method
+* If validated, then "VIES validation passed" field will be False
+* If not validated, then a ValidationError will be shown to user
+
+When VIES VAT Check is not activated:
+
+* "VIES validation passed" field will be always False
+
+
+Usage
+=====
+
+You must preffix VAT with country code (ISO 3166-1 alpha-2) and if you want to
+bypass country validation you can use "EU" code
```

Comment:
This is the standard behavior: https://github.com/odoo/odoo/blob/8.0/addons/base_vat/base_vat.py#L95